### PR TITLE
Edits -- documentation home page

### PR DESCRIPTION
**Pull request by shmcgavin**
@francopestilli 
Quick review of edits:
- Trimmed these descriptions way down -- users will learn plenty more about each aspect as they work with the platform/rest of the documentation.
- Tried to add some more engaging language
- Rearranged sections to follow order of tool bar in platform
- Added new features
- Removed image of old poster & references (references can live somewhere else, maybe home page or a different page of documentation but don't really belong here with instructions unless you want them here)

Also... did I accidentally bold everything or is that just Github?

Please edit for accuracy & style as you see fit

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -2,33 +2,44 @@
...
 
-Brainlife also provide mechanisms to publish all research assets associated with a scientific project (data and analyses) embedded in a cloud computing environment and referenced by a single digital-object-identifier (DOI). The platform is unique because of its focus on supporting scientific reproducibility beyond open code and open data, by providing fundamental smart mechanisms for what we refer to as “Open Services.” 
+Below, you’ll learn about the main panel of tools you’ll encounter as soon as you log into brainlife.io. While these run-throughs are brief, you’ll learn much more about each feature as you review the rest of brainlife’s documentation.
```
Comment (francopestilli) [inline docs/index.md]: @shmcgavin going over the
```suggestion
Below, you will learn about the main panel of tools you will encounter as soon as you log into brainlife.io. While these run-throughs are brief, you’ll learn much more about each feature as you review the rest of brainlife.io’s documentation.
```

Comment (francopestilli) [reply docs/index.md]: Hi @shmcgavin thanks for this! I would prefer if we could avoid contracted forms (we'll or you'll etc.)

File: docs/index.md [hunk 2]
```diff
@@ -2,33 +2,44 @@
...
+
+If you have any questions, don’t forget to reach out to us on the [brainlife.io slack channel](https://brainlife.io/docs/contact/)!
```
Comment (francopestilli) [inline docs/index.md]:
```suggestion
If you have any questions, do not forget to reach out to us on the [brainlife.io slack channel](https://brainlife.io/docs/contact/)!
```

File: docs/index.md [hunk 3]
```diff
@@ -2,33 +2,44 @@
...
 
 # Brainlife Apps
 
-Brainlife uses Apps to analyze data. Apps are small programs, small modules or compute units, that can process data individually or be made part of a larger series of steps in a full data analysis workflow. Brainlife Apps are meant to do small but meaningful steps in a longer analysis pipeline. Apps are modules and the brainlife.io platform. The platform allows users to develop, use, combine, and reuse Apps to build complex pipelines for customized brain data analyses. Most Apps indeed do only one thing, they process data in a specific way and are meant to perform a small set of operations and handle small sets of data; they do one thing, they do it well.
+brainlife.io uses Apps to analyze data. Apps are small programs that can process data individually or be made part of a larger series of steps in a full data analysis workflow.
+
+Anyone can develop, use, combine, and reuse Apps to build complex pipelines for customized brain data analyses on the platform. You can publish apps to be used privately or shared publicly with the brainlife.io community.
```
Comment (francopestilli) [inline docs/index.md]:
```suggestion
Anyone can develop, use, combine, and reuse Apps to build complex pipelines for customized brain data analyses on the platform. You can publish apps to be used privately or shared publicly with the brainlife.io community. Combining multiple Apps allows for high-throughput data processing and aggregation across thousands of datasets.
```

File: docs/index.md [hunk 4]
```diff
@@ -2,33 +2,44 @@
...
+
+# Brainlife Projects
+
+The best way to manage your data is with brainlife’s Projects feature. Here you can organize your datasets, perform data processing, and share results with your team by granting varying levels of access to members of your group. Projects can be public or private.
```
Comment (francopestilli) [inline docs/index.md]:
```suggestion
The best way to manage your data is with brainlife.io’s Projects feature. Here you can organize your datasets, perform data processing, and share results with your team by granting varying levels of access to members of your group. Projects can be public or private.
```

Comment (francopestilli) [reply docs/index.md]: @shmcgavin I think we tend to use brainlife.io not brainlife. Am I wrong?

File: docs/index.md [hunk 5]
```diff
@@ -2,33 +2,44 @@
...
+
+# Brainlife Publications 
 
-Apps can be developed and published on the Brainlife platform by anyone. App developers can be computational neuroscientists, cognitive neuroscientists, but also computer scientists or engineers. Apps are snippets of code implementing algorithms or analyses. By following a few easy steps developers can publish their code for brain data analysis on the Brainlife platform as an App. Publishing code as Apps allows scientists to use the code in combination with the data and computational resources available through Brainlife. Apps published on Brainlife can be used privately or shared publicly with the platform users community.
+Once you finish processing your datasets and producing your data derivatives, you can publish them. A new publication creates a snapshot of your datasets and is archived on brainlife.io for at least 10 years. Anyone can access your publication page and download your datasets without logging into brainlife.io.
```
Comment (francopestilli) [inline docs/index.md]:
```suggestion
Once a Project is mature, the data is analyzed and all the derivatives generated, a portion of the Project can be made public via a brainlife.io publication. A brainlife.io publication creates a snapshot of your datasets and Apps archived on brainlife.io for at least 10 years. Anyone can access your publication page and download your datasets without logging or reuse the Apps and data on brainlife.io.
```

File: docs/index.md [hunk 6]
```diff
@@ -2,33 +2,44 @@
...
+# Brainlife Resources
 
-![ohbm2019](/docs/img/ohbm_poster_brainlife.10dpi.png)
+We know it takes a lot of computing power to run workflows. That’s why brainlife allows users to work with data and computers across a mix of cloud systems and high-performance computing clusters (HPC). Both brainlife users and compute resource providers can register a compute resource that is publicly available to the entire brainlife community or privately available to certain users.
```
Comment (francopestilli) [inline docs/index.md]:
```suggestion
We know it takes a lot of computing power to run workflows. That is why brainlife.io allows users to work with data and computers across a mix of cloud systems and high-performance computing clusters (HPC). Both brainlife.io users and compute resource providers can register a compute resource that is publicly available to the entire brainlife.io community or privately available to certain users. 
```

File: docs/index.md (hunk 1 above)
Comment (shmcgavin) [reply docs/index.md:7]: @francopestilli sorry,  I wasn't sure if I was supposed to resolve these edits but sounds good! I'll making sure not to use contractions as I keep working through documentation.

File: docs/index.md (hunk 4 above)
Comment (shmcgavin) [reply docs/index.md:19]: @francopestilli yes, but sometimes when I add the possessive apostrophe and "s" it looks a little messy. But if you don't like that change, I'll remember to always keep the ".io"

**Comment by shmcgavin** [comment]
@francopestilli I will let you close the PR in case you had any other concerns

**Pull request merged**